### PR TITLE
add plugin vela

### PR DESCRIPTION
**Pull request by sunny0826**
Plugin has been verified.

### CODE REVIEW

File: plugins/vela.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    sha256: 430d0134fb4dc22c36fd3a129a0571796d8c84d6c3d14845a1ef2b1a7584d989
+    files:
+    - from: "*/kubectl-vela.exe"
+      to: "."
+    - from: "*/LICENSE"
+      to: "."
+    bin: "kubectl-vela.exe"
+  shortDescription: A Modern Application Deployment System Based on Kubernetes and OAM.
+  homepage: https://kubevela.io
+  description: |
```
Comment (ahmetb) [inline plugins/vela.yaml:46]: This is not very descriptive. It sounds vague and sales-talk a bit.

File: plugins/vela.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        arch: amd64
+    uri: https://github.com/oam-dev/kubevela/releases/download/v1.0.3/kubectl-vela-v1.0.3-windows-amd64.zip
+    sha256: 430d0134fb4dc22c36fd3a129a0571796d8c84d6c3d14845a1ef2b1a7584d989
+    files:
+    - from: "*/kubectl-vela.exe"
+      to: "."
+    - from: "*/LICENSE"
+      to: "."
+    bin: "kubectl-vela.exe"
+  shortDescription: A Modern Application Deployment System Based on Kubernetes and OAM.
```
Comment (ahmetb) [inline plugins/vela.yaml:44]: Please shorten and be more concrete.

**Comment by ahmetb** [comment]
Are you a maintainer of vela project? If not, we prefer that the maintainers of the project submit it so that they can also pay attention to packaging and sign up for maintaining versions here as well as setting up release automation.

**Comment by sunny0826** [comment]
> Are you a maintainer of vela project? If not, we prefer that the maintainers of the project submit it so that they can also pay attention to packaging and sign up for maintaining versions here as well as setting up release automation.

Yes, I am the maintainer of vela.  After configuring the merge, I will configure `krew-release-bot` to the release action.

**Comment by sunny0826** [comment]
I refer to the ingress-nginx description.

**Comment by sunny0826** [comment]
You can find me in https://github.com/oam-dev/kubevela/blob/fd44069a219e5efad06c42a991df7876e1d7ea87/OWNERS_ALIASES#L11

I am also responsible for maintaining the [brew installation scrip](https://github.com/Homebrew/homebrew-core/pull/66283).

**Comment by ahmetb** [comment]
> I refer to the ingress-nginx description.

you don't have to do that. ingress-nginx is a well known project in the ecosystem (an official Kubernetes sub-project) so this argument does not apply well here. If you have a better description, this would be a good time to fix it. We show them at https://krew.sigs.k8s.io/plugins/.

**Comment by sunny0826** [comment]
Indeed, I have updated the project description, please review.

**Comment by ahmetb** [comment]
I am assuming this plugin already imports client-go auth plugins. let me know if not. Looks good for now.

/lgtm
/approve
/hold

File: plugins/vela.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        arch: amd64
+    uri: https://github.com/oam-dev/kubevela/releases/download/v1.0.3/kubectl-vela-v1.0.3-windows-amd64.zip
+    sha256: 430d0134fb4dc22c36fd3a129a0571796d8c84d6c3d14845a1ef2b1a7584d989
+    files:
+    - from: "*/kubectl-vela.exe"
+      to: "."
+    - from: "*/LICENSE"
+      to: "."
+    bin: "kubectl-vela.exe"
+  shortDescription: Easily interact with KubeVela.
```
Comment (ahmetb) [inline plugins/vela.yaml:44]: can you remove trailing period

Comment (sunny0826) [reply plugins/vela.yaml]: fixed

**Comment by ahmetb** [comment]
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**